lib/components/input-group.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div :class="classObject">
+        <slot name="left"></slot>
```

Comment:
Or I suppose it could be combined for both.  simple usage by the left/right propes, or slots for more complex add-ons.